osbs/api.py

```diff
@@ -726,10 +734,43 @@ def import_image(self, name):
 
     @osbsapi
     def get_token(self):
-        return self.os.get_oauth_token()
+        if self.os.use_kerberos:
+            return self.os.get_oauth_token()
+        else:
+            if self.os.token:
+                return self.os.token
+
+            raise OsbsValidationException("no token stored")
 
     @osbsapi
-    def login(self, token):
+    def login(self, token, username, password):
```

Comment:
This method signature looks wrong. Currently it requires 3 parameters. I think all of them are optional though.

Perhaps instead it should be "login(self, token=None, username=None, password=None):".